Projects/Claims/Sources/Views/SubmitClaim/SubmitClaimAudioRecordingScreen.swift

```diff
@@ -0,0 +1,353 @@
+import AVFAudio
+import Combine
+import Foundation
+import SwiftUI
+import hAnalytics
+import hCore
+import hCoreUI
+import hGraphQL
+
+struct EmbarkRecorderTracking {
```

Comment:
This should probably not be called `Embark*`, Embark is our old flow system.

Reply:
Agree, this was a mistake